### PR TITLE
Tests for json object

### PR DESCRIPTION
**Pull request by mikhailZakharevich**
use created JsonObject  arbitraries in tests

### CODE REVIEW

**Review by VLADISLAV008 — APPROVED** [review]

File: src/test/scala/group/scala/karazin/circe/literal/extras/PlainPrimitiveEncodeSuite.scala [hunk 1]
```diff
@@ -3,17 +3,31 @@ package group.scala.karazin.circe.literal.extras
...
 import org.scalacheck.Prop._
+import group.scala.karazin.circe.literal.extras.arbitraries.instances.{given, _}
 
 class PlainPrimitiveEncodeSuite extends munit.ScalaCheckSuite:
 
   override def scalaCheckTestParameters =
     super.scalaCheckTestParameters
       .withMinSuccessfulTests(1)
 
+  property("raw json parsing") {
```
Comment (IgorWolkov) [inline src/test/scala/group/scala/karazin/circe/literal/extras/PlainPrimitiveEncodeSuite.scala]: Please follow the concept of the tests:
- should be two tests
- follow the structure
- follow the naming
- put the tests to the bottom of the file

**Review by IgorWolkov — CHANGES_REQUESTED** [review]
Please review the existing tests for any primitive type and follow the approach.

File: src/test/scala/group/scala/karazin/circe/literal/extras/PlainPrimitiveEncodeSuite.scala (hunk 1 above)
Comment (mikhailZakharevich) [reply src/test/scala/group/scala/karazin/circe/literal/extras/PlainPrimitiveEncodeSuite.scala]: Done

File: src/test/scala/group/scala/karazin/circe/literal/extras/PlainPrimitiveEncodeSuite.scala [hunk 2]
```diff
@@ -1253,3 +1253,109 @@ class PlainPrimitiveEncodeSuite extends munit.ScalaCheckSuite:
     }
 
   }
+
+  property("inlined java.util.UUID value") {
```
Comment (IgorWolkov) [inline src/test/scala/group/scala/karazin/circe/literal/extras/PlainPrimitiveEncodeSuite.scala]: Haven't we had tests for `UUID` already?

**Review by IgorWolkov — COMMENTED** [review]
Test for `UUID`? Have I missed smth?

**Review by IgorWolkov — APPROVED** [review]

**Pull request merged**